src/AppBundle/Controller/EventController.php

```diff
@@ -199,6 +199,7 @@ public function showAction(JobHandler $jobHandler): Response
             $forms[$formType] = $form->createView();
         }
 
+        $filtersMissing = 0 === $this->event->getNumParticipants() && 0 === $this->event->getNumCategories(true);
```

Comment:
There are 7 equals signs on this line, not readable if you ask me.

Reply:
:) yes it does look a bit odd doesn't it. I've changed it to use implicit casts.

Reply:
I would probably put this in the `Event` class, and just reference it in the view as with `event.filtersMissing`. This is because it sort of ties into `Event::isValid()`.

On that note I wonder if we should hold off on this until #217 is merged, which changes some of this logic.